netlify-webhook-deploy.php

```diff
@@ -348,20 +394,44 @@ public function create_plugin_settings_page() {
 
             add_menu_page( $page_title, $menu_title, $capability, $slug, $callback, $icon, $position );
         }
+
+        if ( current_user_can( $run_deploys ) ) {
```

Comment:
This should be changed to:
```suggestion
        if ( current_user_can( $adjust_settings ) ) {
```
Otherwise, everyone with permissions to perform deploys will see the new "Schedule Build" settings.

Alternatively, you can introduce a separate filter for this. Something along the lines of:
```
        $schedule_builds = apply_filters( 'netlify_schedule_builds_capability', 'manage_options' );
```
But I think the first option (reusing $adjust_settings) should be sufficient.

Reply:
Committed your suggestion. Nice one!